bait-files/src/main/java/com/google/sps/servlets/Address.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.servlets;
+
+public final class Address {
+  String addressLine1;
+  String addressLine2;
+  String city;
+  String postalCode;
+  String zipCode;
+  String countryCode;
+  String state;
+  String province;
```

Comment:
private final?

Reply:
Making them final would mean I would not be able to replace the default address object with the properly parameterized address.